keen/api.py

```diff
@@ -194,12 +183,13 @@ def get_collection(self, event_collection):
 
         return response.json()
 
+    @requires_key(KeenKeys.READ)
```

Comment:
Note the change in KeenApi.get_collection()/get_all_collections() to relax the required key from `master_key` to `read_key` could technically break clients. If a client was setting only the `master_key` and only calling methods that needed a `master_key` including one or both of those methods, they'd now fail.